rules/application/antivirus/av_ransomware.yml

```diff
@@ -1,12 +1,12 @@
 title: Antivirus Ransomware Detection
 id: 4c6ca276-d4d0-4a8c-9e4c-d69832f8671f
-status: experimental
+status: test
 description: Detects a highly relevant Antivirus alert that reports ransomware
 references:
     - https://www.nextron-systems.com/?s=antivirus
 author: Florian Roth
 date: 2022/05/12
-modified: 2021/12/02
+modified: 2022/12/25
```

Comment:
Can't promote this yet. It's only been 6 months since creation.

Reply:
created in 2022 but modified in 2021 .
Original  `date: 2018/09/09` in the commit history

Reply:
modified was 2021 ... 
I did not make a test to check when modified is older than data